docs/data/joy/components/alert/alert.md

```diff
@@ -27,28 +36,57 @@ export default function MyApp() {
 }
 ```
 
+### Basics
+
+The Alert component wraps around its content:
+
+```jsx
+<Alert>This is a time-sensitive message.</Alert>
+```
+
+### Anatomy
+
+The Alert component is composed of a single root `<div>` element with its `role` set to `alert`:
+
+```html
+<div role="alert" class="JoyAlert-root">
+  This is how an Alert renders in the DOM.
+</div>
+```
+
+### Overriding the root slot
+
+Use the `component` prop to override the root slot with a custom element.
+For example, the following code snippet replaces the default `<div>` with a `<span>`:
+
+```jsx
+<Alert component="span" />
+```
+
+## Customization
+
 ### Variants
 
-The alert component supports the four global variants: `solid` (default), `soft`, `outlined` and `plain`.
+The Alert component supports the four [global variants](/joy-ui/main-features/global-variants/): `solid` (default), `soft`, `outlined`, and `plain`.
 
 {{"demo": "AlertVariants.js"}}
 
 ### Colors
 
 Every palette included in the theme is available via the `color` prop.
-Play around combining different colors with different variants.
+The demo below shows how the values for the `color` prop are affected by the global variants:
 
 {{"demo": "AlertColors.js"}}
```

Comment:
A side note, this demo overflowd on mobile 

![Screenshot_20221025-210719_Chrome](https://user-images.githubusercontent.com/3165635/197860522-879fdf13-6ad8-4282-8dac-86515234f037.jpg)